pkg/controllers/user/workload/workload_common.go

```diff
@@ -620,7 +620,7 @@ func getWorkloadID(objectType string, namespace string, name string) string {
 	return fmt.Sprintf("%s:%s:%s", objectType, namespace, name)
 }
 
-func (c CommonController) UpdateWorkload(w *Workload, annotations map[string]string) error {
+func (c CommonController) UpdateWorkload(w *Workload, annotations map[string]string, podTemplate *corev1.PodTemplateSpec) error {
 	// only annotations updates are supported
```

Comment:
Apparently, the changes break this comment. I wonder if there was any concern about this. Otherwise, update it.